py-test/run_tests.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Run unit tests for Contact Line Subgrid Modeling
+# This script runs all pytest tests in the current directory
+
+echo "Running Contact Line Subgrid Modeling Tests..."
+echo "============================================"
+
+# Add parent directory to PYTHONPATH so tests can import the modules
+export PYTHONPATH="${PYTHONPATH}:$(dirname $(pwd))"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)